README.md

```diff
@@ -1,56 +1,90 @@
+work together in a team setting on the same sets of files? Your filenames get very
+verbose at times: 
 
 ```
-~ $ mkdir next-big-thing
-~ $ cd next-big-thing
-next-big-thing $ ls
-
+important_presentation-v1.docx
+important_presentation-v2.docx
+important_presentation-v3.docx
+important_presentation-final.docx
+important_presentation-final-final.docx
+important_presentation-final-final-FINAL.docx
+important_presentation-final-final-FINAL-SERIOUSLY.docx
 ```
+Luckily, there's a better way to keep track of versions of files and share them amongst 
+multiple parties without causing complete chaos or confusion.
 
-You created a new directory with `mkdir next-big-thing` and then moved into it by changing your directory with `cd next-big-thing`. You typed `ls` to see all the files and folders in your brand new project, and, as expected, it was empty. But this directory, `next-big-thing`, is where you're going to put all your code.
+## Define Git and explain how it helps programmers control and manage changes
 
-The first step is to transform this directory into a Git-enabled directory, which will allow it to keep track of all the changes to your code and interact with other Git remotes such as GitHub. How do we accomplish this? It's as simple as `git init`.
+If you have a project that you've already started building on your local machine, 
+you will want to be able to start tracking your changes and storing these changes 
+somewhere accessible so that other developers can review or collaborate on the project--
+or just put it somewhere you can come back to it later, regardless of what may happen 
+to the physical files on your computer. This is where `git` comes in! `Git` will allow 
+the users to keep track of these changes and keep detailed tabs and backups of your project.
 
-`git init` initializes a new Git repository in your current directory. Type `git init` from within your `next-big-thing` project directory.
+Now that we've established what `git` does and how useful it can be, how 
+can we start using it? I you haven't already generated some files on your local machine at 
+this point--You can do so by using these commands to create a project:
 
 ```
-next-big-thing $ git init
-Initialized empty Git repository in /Users/avi/next-big-thing/.git/
+~ $ mkdir my-git-project //Creates new directory
+~ $ cd my-git-project //Moves into the newly created directory
 ```
 
-After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/next-big-thing/.git/`. Git is letting you know that it created a new repository within the hidden `.git` folder in `next-big-thing`. This hidden directory, `.git`, is what Git uses to keep track of all the different versions of your code.
+The files are available now, so how do we get them in the cloud so we can collaborate with 
+other developers or backup your progress thus far? 
+
+## State initializing Git repository with `git init`
```

Comment:
Use [AP style headings](https://github.com/learn-co-curriculum/curriculum-team/blob/master/style_guide.md#headings_should_be_capitalized_according_to_ap_style)

Try to follow the style guide as well. It's a lot to hold, but you'll get the hang of it.